src/torrent_info.cpp

```diff
@@ -1475,8 +1475,8 @@ namespace libtorrent {
 				web_seed_entry ent(maybe_url_encode(url.string_value().to_string())
 					, web_seed_entry::url_seed);
 				if ((m_flags & multifile) && ent.url[ent.url.size() - 1] != '/') ent.url += '/';
-				if (unique.count(ent.url)) continue;
-				unique.insert(ent.url);
+				auto const uniq = unique.insert(ent.url);
+				if (uniq.second == false) continue;
```

Comment:
I wonder if it would make sense to not even name ``uniq`` here, like this:

``if (!unique.insert(ent.url)) continue;``